core/src/main/java/feast/core/job/Runner.java

```diff
@@ -27,10 +27,15 @@
     this.name = name;
   }
 
+  /**
+   * Get the human readable name of this runner. Returns a human readable name of the runner that
+   * can be used for logging/config files.
```

Comment:
I think your approach should be the other way around. As a best practice, it's not a good idea to make your application logic depend on `toString()` . Use `toString()` for the purpose of being read by human

Reply:
The codebase currently favors using `getName()` for human readable applications like logging and config. However, I think that your point about  `toString()` is valid as we don't want to end in a situation where our code breaks because someone overwrote `toString()` to make it human readable.

Hence I propose that we:
- replace `getName()` with `toString()`
- use `name()` when converting to string  for Job.runner as it cannot be overridden
> Reference: https://stackoverflow.com/questions/13291076/java-enum-why-use-tostring-instead-of-name